Add MongoDB recipe (#20)
redskyops/redskyops-recipes: mongodb/README.md

`kustomization.yaml`
: The input to Kustomize used to build the Kubernetes object manifests for this example.  

`application/results-exporter.yaml`
: The manifest for the results exporter (a utility container that holds results for the experimant loop).

`service-account.yaml`
: The manifest for the service account used by the controller to create and manage all of the assets in the experiment. This is applied just once to create the kubernetes user, role, and rolebinding.

`mongo-perf/`
: This directory contains files required to run the lightly modified mongo-perf container. There is *no need to build* the image; the folder is only included for reference.  

`mongo-perf/benchrun.py`
:The modification of this file is simply allowing the results JSON files to be kept and persisted in the result-exporter(below).

> `result-exporter/app.py` :Contains the exporter application, `requirements.txt` are the python library requirements and `Dockerfile` contains the Docker commands to run the exporter container. There is *no need to build* the image; the folder is only included for reference.

Reviewer: ```suggestion
`result-exporter/app.py`
: Contains the exporter application. The `requirements.txt` file outlines the python library requirements and the `Dockerfile` contains the Docker commands to run the exporter container. There is *no need to build* the image; the folder is only included for reference.
```

> `result-exporter/` : This directory contains files required to run the results exporter. The exporter's role is to expose the throughput metric over HTTP (in contrast to the original websocket implementation in the results service) so that it can be collected by the jsonpath metric collector.

Reviewer: ```suggestion
`result-exporter/` : This directory contains the files required to run the results exporter. The exporter exposes the throughput metric over HTTP (in contrast to the original websocket implementation in the results service) so that it can be collected by the jsonpath metric collector.
```

> `result-exporter/app.py` : Contains the exporter application, `requirements.txt` are the python library requirements and `Dockerfile` contains the Docker commands to run the exporter container. There is *no need to build* the image; the folder is only included for reference.

Reviewer: ```suggestion
result-exporter/app.py` : Contains the exporter application. The `requirements.txt` file outlines the python library requirements and `Dockerfile` contains the Docker commands to run the exporter container. There is *no need to build* the image; the folder is only included for reference.
 ```

## Experiment Lifecycle

> For the experiment, the pods and service for the result-exporter will persist throughout the entire experiment.

Reviewer: ```suggestion
Throughout the entire lifecycle of the experiment, the pods and service for the result-exporter will persist. 
```

> For every trial, several pods will come up, as needed for the testing.

Reviewer: ```suggestion
For each trial, the following pods will come up:
```

1. `to create the experiment asset. Also has an init container to wait for all pieces to spin up.`
2. `mongo-db itself, spun up by Helm`
3. `trial execution with the mongo-perf container`
4. `trial delete pod to cleanup after the trial`

> 

Reviewer: can we add what the name of the pods might look like?
